File: CPP/Program/perusahaan.cpp
#include <iostream>
#include <iomanip>
#include <string>
using namespace std;

class Perusahaan {
private:
    string pemilik;
    string nib;
    string lokasi;

public:
    Perusahaan(string pemilik, string nib, string lokasi) {
        this->pemilik = pemilik;
        this->nib = nib;
        this->lokasi = lokasi;
    }

    // Setter & Getter
    void setPemilik(string pemilik) { this->pemilik = pemilik; }
    string getPemilik() { return pemilik; }

    void setNib(string nib) { this->nib = nib; }
    string getNib() { return nib; }

    void setLokasi(string lokasi) { this->lokasi = lokasi; }
    string getLokasi() { return lokasi; }
};

File: Python/Program/Perusahaan.py
class Perusahaan:
    def __init__(self, pemilik, nib, lokasi):
        self._pemilik = pemilik
        self._nib = nib
        self._lokasi = lokasi

    # Setter
    def setPemilik(self, pemilik): self._pemilik = pemilik
    def setNib(self, nib): self._nib = nib
    def setLokasi(self, lokasi): self._lokasi = lokasi

    # Getter
    def getPemilik(self): return self._pemilik
    def getNib(self): return self._nib
    def getLokasi(self): return self._lokasi
